azure-pipelines.yml

```diff
@@ -83,10 +83,10 @@ stages:
         yarn bootstrap
         set CI=false
         yarn build
-      name: Build
+      name: Bootstrap
     - script: |
         set CI=true
-        yarn test
+        yarn test:ci
```

Comment:
Thanks to this commit that I've found something interesting.

We're testing `*.ts` then why do we build before the tests?